src/defaults/collections.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "name": "Nitnem",
+    "id": "0",
+    "type": "bani",
```

Comment:
Technically, `Nitnem` is a folder, right?

Reply:
yes. technically for a `folder` the type really doesn't  matter cause it never gets used. so anything in here will work at least for now. In future when we implement a folder then `type` will be more than `ContentTypes` we have it will be a unions which then can help is fetch the folder information accordingly.

Reply:
In that case, worth removing or nulling `type` here, to avoid confusion